src/Common/Core/Impl/Shell/IApplicationConstants.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.Common.Core.Shell {
+    /// <summary>
+    /// Defines application constants such as locale, registry key names, etc.
+    /// Implemented by the host application. Imported via MEF.
+    /// </summary>
+    public interface IApplicationConstants {
```

Comment:
ICoreShell is getting bigger.... Almost 20 methods/properties. I want to split it into constants, shell related methods (show message box, progress, idle, app lifetime events) and basic/core services.